package.json

```diff
@@ -229,9 +229,8 @@
   },
   "scripts": {
     "build": "cross-env NODE_ENV=production webpack --config webpack_config/production.js",
+    "build:staging": "cross-env NODE_ENV=staging webpack --config webpack_config/staging.js",
```

Comment:
Staging should have the same build config as production.
Prefer
`build:staging": "cross-env NODE_ENV=staging webpack --config webpack_config/production.js`